shared/actions/teams.js

```diff
@@ -594,12 +591,12 @@ const _getTeamPublicity = function*(action: TeamsGen.GetTeamPublicityPayload): S
   )
 
   let tarsDisabled = false
-  // Find out whether team access requests are enabled. Throws if you aren't admin.
-  if (yourOperations.changeTarsDisabled) {
```

Comment:
this logic is incorrect. i get maybe your operations could be not loaded yet or something? anyways, lets just load this and catch the error if it happens. else you can never load it (in my testing)